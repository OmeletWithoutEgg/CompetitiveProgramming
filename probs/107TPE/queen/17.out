1194195.83 8036855.05
