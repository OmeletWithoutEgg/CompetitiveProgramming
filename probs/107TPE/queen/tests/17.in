1542
-67044105 78185444
119768480 -21876626
49635972 -86190499
-99103654 -16365850
-41294330 8961292
-78449759 -17640938
-77788064 -17538578
27064232 -26347034
42074229 -80922983
-53507160 70026069
120572002 -61680567
-42717272 6383026
71787095 7231842
-77151691 -17431550
108000003 -5755964
-118857309 -5658327
-41710126 8169952
93897198 -88709278
-118107008 -6307711
94660968 -88376600
-123534454 -939484
35328206 -11396309
120646232 -61498596
26089948 -55915839
-77286624 -17454950
123486795 -51755878
112432683 -10348728
32686336 -15005458
-71707550 -16162535
-76310407 80919874
-134587664 42596368
-130532210 54085054
-36078152 37431176
-98193221 80251532
33082499 -14419132
-101727109 79204219
-54422232 70780240
53070350 2558833
-118470085 69639551
92155025 -89412557
26290956 -28779872
-115943271 71685167
49001023 -85824006
-82284640 -18057485
-115658370 71899273
-77396759 -17473767
76351082 7254123
119597476 -63901244
-52689249 69320250
-72375386 79995955
89142109 5046026
-41198998 9147692
109173149 -78547936
-132822788 48713001
-88539416 -18101410
-46251753 1179876
-115452856 -8408927
44141580 -2833405
123927729 -48898786
24390909 -39912898
121680411 -26694273
124310143 -43232008
-58504853 73738271
-75779275 -17171808
105400437 -3543538
99991188 -85606847
-126122066 2306973
55768619 -89138235
-35887665 28285222
40945609 -5466944
-135395087 25771419
-106468561 77332485
-102651298 -15239898
-131372471 11334175
-36062728 26349988
-84927679 -18171615
-35814508 34102193
-52047593 68744412
-36834303 21524070
-40954256 54007553
-38809867 49010187
-54528119 -7222353
94760053 2923689
120799128 -61117171
87368523 5561108
123947125 -36667228
65048146 -91838460
-135180366 39428698
-58729151 -10240445
53852182 2917583
-121551947 66737210
-117439620 -6863951
-95038208 -17310657
102170073 -1183487
114438085 -12871924
-55988810 -8347349
-82437937 -18067937
-70745248 79512959
-65970385 -14094526
25814677 -30542094
29369709 -20792972
-110277648 -11756166
-79085273 -17730730
-44883231 60611839
-93299132 -17607307
-78552036 -17655951
-107233834 76976483
123979208 -48471515
51052561 -86964406
97639679 -86929455
27738858 -60893766
-45702897 61741831
-46097613 1379671
94563305 -88419985
-38732267 14846116
-119535596 68691124
-110302969 -11741912
107659282 -5448170
101102724 -486943
-115327641 -8501037
-41815763 7974242
109827744 -77899457
-40507699 10560971
-66431716 77933478
122714937 -55244074
-119402848 68812285
-38083332 46877997
38637996 -7671487
29578595 -20369604
-135410555 37743137
118186691 -66685412
-69264192 79021067
-90550870 81591599
122213182 -28366760
119998800 -22386812
-102219856 79035335
104605524 -82483190
62218051 -91219409
-36356234 39504171
-111810048 -10857892
-122205040 66055003
-66704011 78046684
-134210629 19461195
-52362715 -5386876
47952660 -217423
-35895759 28172861
25939622 -55354351
-105742783 -14012925
-114262563 72903580
65470668 6507616
-44575401 60169714
-126498324 60814338
-132441036 49740976
-39411452 13070843
47257378 -656759
-65117019 -13717202
114495088 -12948748
-133093897 47937717
78691951 7103554
124225221 -39744088
24690790 -48839743
-96812654 80585062
-47267003 63729090
-65858702 -14046224
109825471 -77901751
-82358557 81705415
-91320267 -17868732
-35768672 32609442
-97311467 80469339
-124720228 479871
-46427369 954759
99431385 528344
71176458 -92605847
-39097994 13865151
84134406 -91730215
32800410 -70573852
122047390 -57584543
75048080 -92698914
123595908 -51141609
-132485227 49625438
25787262 -30651940
67704606 -92265695
118103119 -18570957
-133592814 46390473
105108423 -3313489
-36731357 22025969
-97568111 80407696
-122090934 66176067
110694992 -8406427
-50179652 -3304980
-53648188 -6501860
-51519011 -4611894
-47284761 -107671
-46568295 62866823
-53690522 -6537296
-57767811 73249662
106070533 -4084965
-55131607 -7697577
-132679893 49105936
102664722 -83888972
30796298 -18081099
-104002421 78375727
-82077180 -18042587
65028366 6426089
-44056484 4242230
-37945179 17209520
-38377483 47779605
-39764110 12221628
68864798 6997974
47451960 -532190
31014437 -17699504
-106288983 -13770930
-114848575 72490790
123588992 -51181907
124099015 -47324435
29057001 -63962285
-74480459 80531952
-51913475 68621499
-77031583 -17410401
27921315 -24054430
-95714022 -17178242
-134932527 40893604
-128868183 6485733
92473667 3880953
-73498656 80294079
-37466706 44764367
123829283 -49641934
-93675734 -17548415
124306193 -41883712
123789546 -35488767
-111739684 74543587
-38021123 46679565
39661880 -6658447
96395601 2154968
-37101213 20328682
115408459 -14223495
-35766659 31173533
-44711520 60366471
-51677482 68403026
115014573 -71745256
112217656 -10097087
49477419 691505
106774569 -4674832
73486755 7288855
115553925 -70974092
63871102 -91601847
-120249949 68023807
28770492 -63341338
-132101487 50601585
-90732069 -17931002
-92423711 81375707
115916320 -14970625
25824860 -30501560
-36452133 23545455
-41232763 9081451
85937745 -91334182
-80214526 81512666
122433439 -56284317
75193952 7287919
123898272 -36278308
114509467 -72440462
-71061250 79611314
-102842748 -15170752
29543994 -64969596
-50299578 -3426051
-129853568 55400523
-61368932 -11824383
-36099862 37620228
-134780657 21961571
93045978 -89062412
-134636461 42372820
123061492 -31587778
24516215 -47208480
-133362647 47125741
24315523 -43213766
85428162 6044544
-45753881 1832859
-128153842 58335014
-48756724 65445446
-94275882 -17448471
123032101 -53948969
28940898 -63713340
25353780 -52853304
-76045015 -17225207
29506535 -20514519
-132937283 48390551
-85360829 81819802
95737492 -87881614
44527596 -82864423
-118199759 69871668
-41740609 55529568
-123491131 -989368
-37140651 43479871
-96942129 80555542
-133909248 18334868
99115903 710108
-39385838 13134287
66909274 -92153140
-78388460 -17631836
-133658413 46173344
87872227 -90830667
-46438431 62702165
-55810220 71857190
-35907703 35629703
120701141 -24045977
-128273557 58142638
24461832 -38848373
-89962952 -18001831
-64455192 77054150
-133979916 45053086
-76418968 80940649
104968396 -82204220
-38651059 15072810
-113404158 73485988
-91788025 81457013
-42432054 56770881
-98436810 -16545524
29646275 -65174080
-106095959 -13857342
103350889 -2000550
78743358 -92507645
-124105811 -268887
-111510371 74682180
-41462678 8636739
62063043 5771865
80928508 -92264727
-100130665 79712487
96734037 1986775
101835275 -84447825
-82665712 -18082594
-123729018 -713805
-107509004 76844608
74998321 -92699621
102641236 -83905131
46740306 -994002
-38013592 46655345
28968644 -21635479
44241737 -82650827
-132128257 50535395
25656067 -54215616
-106787737 -13543084
112315397 -75197584
25688990 -54353900
108580224 -6293436
-112015320 74374763
-104258760 -14631646
72650026 -92676658
70890176 -92587029
123208221 -53156606
-45548482 61533969
115734632 -70708647
-125000069 62811901
-132832939 14958138
-40959853 9623978
31394233 -17053088
-106309248 -13761800
101548595 -773203
-126432617 60906437
-57482182 -9412379
39573625 -6743494
-49368947 66107157
-127643311 4508517
27567793 -60449527
-44226078 59655367
-134345696 20003239
61111248 5521356
-124114302 -258740
120707825 -24062606
-118270395 -6168507
98087381 1281763
35270255 -73940031
-125007232 840001
51986564 -87442820
-48377650 -1380631
85561639 -91422554
73754815 -92701204
-103128874 -15065765
124302067 -43746411
-99823966 79803434
-83838519 81784386
31856367 -16295220
32963995 -70815985
-97233591 -16844930
-46448713 927578
-36450499 23555198
-94141942 -17471427
121743099 -26880954
-125830349 61730685
123569817 -34116020
-45951119 62071325
-79008507 -17720326
-49864356 -2982694
-121284595 67009170
31637882 -68758886
-36754819 41733577
113685750 -11885803
-82323489 -18060179
77948052 7163365
-118952303 69217063
35414348 -74119292
-41480111 8603461
55684317 3695842
-133507161 16974070
-127913821 58714963
-51066475 -4181202
42795111 -3889022
114456466 -12896663
119721163 -63634970
-61203706 75374458
99018115 -86174466
24325414 -43821668
59670776 5103708
-103465196 -14939820
-39694816 12385057
110033505 -77690607
-105068128 77943983
113770656 -11994586
-92039846 81425795
56709463 -89503016
-53966322 70409050
107755731 -79873915
-114274383 72895377
-47112668 100863
87881428 5419436
122889306 -54550644
124264241 -40498344
97377190 1658276
59876278 -90574802
-135739204 33346289
-35924572 27798382
124026944 -37364226
-36301663 39144629
-67976799 78551173
86878907 -91099527
-37433123 44638396
78736346 -92508268
85362471 -91468112
-135049351 23406993
122612518 -29775686
74098594 -92703859
-72440024 80013870
40077622 -79145283
66008946 6601299
-55940063 71954002
-135571864 36182997
-128096096 5215713
-109020594 76082677
-125289776 1201301
-53428740 -6316892
-130355713 9206333
-46604057 62911919
80095657 6960156
91960443 4077785
-38830225 14577136
47429888 -84862380
106961954 -80573053
-112099234 74322879
-135759903 32327256
88598162 5211567
-54904818 71163579
49641007 -86193355
-82714561 81728432
123953517 -36720019
104555884 -82520943
-37269136 19639569
-110088812 -11861861
61259212 -90970254
-40475385 10629892
42763873 -81494229
124259095 -45023881
88118412 5351988
40821293 -5578715
-125024107 861389
35841373 -74640810
-64646346 77143702
-103376190 -14973419
122711441 -30151075
24408093 -39620893
-37906367 17337140
38235296 -8086339
-43386358 5283408
90251936 4687115
37542722 -8822879
35904855 -74717128
50954823 -86912903
-130387047 9268226
43914760 -82402573
65024610 6425379
-43636139 58754165
108960100 -6654721
124122150 -38340681
122852152 -54701970
122171475 -28228150
107884440 -79757706
-55376831 71529267
-40021537 11628154
-82087143 81686153
24600601 -37348518
-132282516 50148569
120276544 -23023211
112026200 -75532798
100201518 71620
36081946 -10480227
80903947 6859362
-66251917 77857691
33061004 -70958147
-111977005 74398378
121932023 -57949703
64367035 6296490
59448555 5035083
-50521163 67290429
24318073 -41986983
-57487377 73058772
26052633 -55778873
121782951 -27000915
-84683586 -18166943
55310401 -88952605
124234039 -39896718
-75560500 -17126720
64186854 -91668210
123209307 -32257116
-117665134 -6678189
56797542 -89536052
-130297832 9092562
58685942 4790916
-38250374 47397088
-60041099 74698117
-62395966 76025566
66522684 6685074
-49166329 65890802
56192311 -89305212
87071294 5640527
-133765799 45809969
-83337915 -18119766
29669667 -20188118
58212099 -90041020
-110873416 75058436
30502794 -66801460
67300036 -92210069
-135754477 32714938
45357239 -83466656
122896104 -54522736
-38404965 47860977
24410340 -45823891
93425165 -88907374
31861598 -16286812
-120135466 68132524
93010892 3667937
-133817351 18011000
-135760787 31412272
32636120 -70327565
-126874331 3364181
-127398479 4136725
87942054 -90810939
-135457452 37335752
-114757549 -8912906
24985190 -34532762
97361569 -87075031
34993724 -73591213
-35777524 33048610
115641145 -70846429
-114210486 72939658
111818503 -9638750
-127889687 4890079
25112943 -33795811
-130190738 8883930
123262831 -52897793
48954874 -85796883
-104294798 -14617281
54277688 3106043
-132985603 48252330
70293570 7133867
101467679 -720757
46223394 -84068289
81508320 6775236
30358272 -18871815
26151700 -56139205
36346044 -10169479
-86513712 -18172940
78375707 -92538983
-123479328 64645766
27042217 -58997874
-133749822 45864678
70158661 7122803
-50743695 67510153
44006691 -82472799
-127965360 58634012
-116940215 -7267556
60096827 -90640761
-68971349 78917684
-37314114 19461902
-42098272 56182009
120576286 -23738515
-135373878 25599893
49904537 -86341808
-92092255 -17776303
124161268 -46595719
49182840 521585
-135382953 25672693
86185726 5865597
-35776562 30634062
-105015446 -14323168
-135674884 34779471
-135749831 30697653
44695710 -82988564
-124463794 163765
96313477 2195301
-132299525 13537495
25704984 -30988181
67919274 6885209
-61238534 75394079
-38326202 16016313
95640069 2519050
-77839953 -17546933
38004699 -8328272
71164423 7196452
112635635 -74819270
124307781 -43421690
24503358 -38344636
-105094200 77933061
119033772 -65066028
91516905 -89651344
-129352982 56313589
-106814480 77173510
31103428 -67862601
75218518 7287480
119173433 -64784515
-101479675 -15644037
-111193097 74871187
113251353 -11338748
41711904 -80614387
119268245 -64590831
-38612445 15181825
-76967164 81041764
-122077696 -2547230
36635749 -9834420
122934190 -54365058
107828151 -5600009
39931639 -6401054
29374530 -64625551
109853004 -77873950
95941886 2375479
-76940238 81036945
119023745 -65086070
-115736598 71840800
41237773 -5207197
51059576 -86968092
36721916 -75672725
-124330248 63641626
41944067 -4595831
26939274 -58696097
-133515783 46641006
-134903408 22592800
116865161 -68958984
110907808 -76775050
106811936 -4706760
-123925323 64126152
103977835 -2455179
92795188 3754330
-55154571 71358197
-44075955 59430022
60958741 -90887985
79126113 -92472112
95573799 -87958880
-62584011 76124456
-70096172 79303700
124309754 -43267661
-133616730 46311722
-133967884 45096854
83354093 -91880098
74043956 -92703595
-37708062 45633537
26336183 -28624831
-100369632 -15997311
56405231 3978809
-65253731 77421766
31307239 -17199207
-130559230 9612193
60336565 5302570
-35766018 32417776
120699913 -61365712
120416220 -23352667
30344077 -66510625
-112087474 74330164
120380513 -23267818
-135584725 36033546
-135745794 30530506
72153069 -92657647
-46019632 1481550
-49014734 65727233
61022846 5497069
116494241 -15857777
107603379 -80010429
-68463509 -15090696
-124344261 18197
-135493454 37001032
-76534445 80962475
-110387483 -11694195
25185738 -52005889
120347517 -62218842
-126050116 2208835
-49404743 -2502284
-79047003 -17725558
86926956 5678388
111308620 -9069237
-45568256 2082117
-55117209 -7686413
49986902 979139
-134161755 44371651
89702560 -90276992
76322725 7255272
-106476736 77328765
-133417013 16687226
124129488 -38425264
46551514 -1119418
-68237403 78649530
51152020 1607896
70349971 7138384
-134913098 22644544
120694681 -24029927
-38648372 15080372
-132726993 48977545
-51534690 68269456
112668826 -74779621
121330334 -59715007
-38532834 15409156
25395446 -32356370
73084748 -92689232
52611775 2340770
92109811 4021167
74098165 7295220
-135473623 37188020
122280988 -56813365
-47848757 -775379
112881587 -74523472
52797646 -87838481
79471878 7028822
-40167173 11301434
-131188134 52714197
121836877 -27164877
52228405 2154057
-38813577 49020318
27942908 -61407828
-83581418 -18130992
-49441864 66184393
-84788148 -18169091
-37162004 20074264
-51320549 68067170
89820183 4830143
-133950198 18481925
120297024 -23071110
110558359 -77146570
112195043 -75337819
-37807774 45975846
-122480558 65759320
84561815 6234013
-45118939 2699189
-37029718 43007110
-38500593 15502230
103904779 -2401436
-44302399 59768924
-69554967 -15478871
-96703583 80609650
81908432 6715397
-62713225 -12548988
43188305 -81835608
-61896605 75757971
103407955 -2041321
64619512 -91755697
56389963 3972950
114094310 -72993617
-39722536 51323345
124303254 -43687813
123171070 -32079729
-81539055 -17999894
96628135 -87448385
123773627 -35380438
117070443 -16785312
59811058 5146447
-125398477 1342140
80284440 6937811
51183323 1624243
116024993 -15134342
73867839 7293700
-123168033 -1357241
102709706 -83857970
-49938185 -3058682
121946664 -57903896
-63904280 -13147684
122276237 -28579130
27031045 -58965424
122755678 -55085753
-125760670 61823804
32574574 -15174155
120942766 -24658316
26133034 -29336525
31623633 -16673104
93018887 3664713
-35863655 28641932
94109372 -88618377
-118292834 69792126
117451888 -17425275
-101355634 -15684931
69804363 7091995
104354548 -2735568
82060828 6691734
89924202 4796083
120659503 -61465845
27767485 -24441741
61656616 -91075982
25371632 -52939044
-112269083 74217159
-41775073 55593451
120465229 -61938795
-67178853 78239618
24454339 -38946507
-61121509 -11685176
-131515266 51987747
50177084 1084505
35016018 -73619573
-98457083 80183039
-130758802 10019366
-101804960 79177917
-42102668 7452943
-120472181 67810794
-35832796 34472543
-131747135 12189558
-110295474 75388964
-56154460 -8469604
111560501 -76060251
38150204 -8175237
-56301718 -8577403
90630739 4558024
-66441425 77937546
-122217999 -2398371
-37363677 44373614
24966731 -50763653
-134873691 22435943
-104765499 78069492
-116478160 71274500
26939656 -58697227
-55473811 -7960461
24340174 -41053901
123926088 -48911923
24719964 -36337375
-89239208 -18057560
-37417196 19064640
110044383 -7729140
119940578 -63152239
-111181767 -11235046
62189480 5803640
24721620 -36324492
35652018 -74411322
28030791 -61624251
58629385 4772271
-42423494 6886824
120041497 -62925543
24520042 -47250590
-62540623 -12458898
98501347 -86464122
24355620 -40638552
82700128 6587211
71022676 -92595944
-78140533 -17594232
-72738571 80095407
117830055 -18081980
58008400 -89971245
-39929039 51803842
-105121892 -14278611
-113080721 -10055937
-78148666 81238317
26085245 -55898658
-92078841 -17778013
98607542 -86405254
-123560824 -909057
-84125009 81794596
-125667394 61947756
-100451499 79615057
-111501344 74687602
120391070 -62115778
114200370 -12554826
-135221715 39155056
-36215897 38539896
55922939 -89199569
-38991652 49498840
-132268039 13457557
97955420 -86761470
-66491613 77958541
-40445869 10693083
49770996 -86266844
122579946 -55754028
-110064017 -11875660
-36772897 41822543
-134060690 44755135
40170234 -6176569
-130800010 53538212
-38300889 47550342
95503315 -87991927
-60250443 -11180177
86547714 5775677
51388919 1730914
-121500367 66790003
82347848 6645859
64643188 6351732
-102149308 79059866
-71834863 79842473
-133036077 15536291
-130876608 10264193
-69025916 78937103
98709601 939722
124303108 -41713414
-74716390 -16943165
88339645 -90696518
48535702 -85547458
-41205109 54507146
-100079490 79727811
24861780 -35316219
-46177052 1276443
60201525 5263049
-127322692 59619748
25425718 -32214288
119929526 -22231753
-129109917 56741249
-37384216 19190288
24388334 -39959324
-113990236 73091172
99928462 -85644328
-129676906 7914898
25211486 -52140869
-113085274 73695798
-87880952 81776515
-127712644 59027705
-134797324 41598031
117887876 -67224186
-107407463 76893512
96542168 2082520
-68786048 -15208377
25367450 -32489610
29096631 -21362188
110635295 -77065425
-87886919 81776262
24547181 -47538625
-66243914 77854298
-43845456 59078793
-45537034 2124362
-63530375 -12964010
84361284 6275581
77272013 -92616680
121511295 -59206260
58489482 -90134463
25536932 -31708763
-121430227 -3218711
-106072379 -13867831
108959565 -78754430
67176736 6783809
117974886 -18339726
-134901600 41059654
-61358892 -11818770
78180063 7145910
-36237204 24947632
-124649537 63250628
-105123409 77920805
-134732256 21723930
102130985 -1157289
124035075 -47968162
-69632614 -15505406
111588305 -9379459
48751312 -85676447
-81080748 -17958918
39602868 -78693369
100935056 -381012
-37233054 19784117
-71862495 -16207648
99011881 -86178008
123736684 -35135152
-106609531 77268084
-94565883 81040312
-132506485 14073282
53458106 -88147430
120950678 -60729861
78345631 -92541426
53339159 2684017
-131263731 11093789
34364972 -12635051
-83734207 -18137429
-123456905 -1028683
24475717 -46735753
-88338655 81755004
-135678993 34709294
-77659680 81160497
-36801168 21682790
122129962 -57317025
62481620 5875720
-135364315 38118727
72281956 7254416
26241551 -28951400
56610680 -89465739
40902831 -5505321
-99676216 -16203655
112662165 -10621052
-38457879 15626462
-92103523 81417695
73547317 7289820
105373651 -3522289
58492961 4726986
-78580726 81302954
-64865924 77245360
-105933633 77572086
-133578256 46438179
91896825 -89510371
30381878 -66580308
-134960050 40743146
-95382170 -17244471
-99334839 79944077
107863939 -79776271
-52654867 69289898
-130324989 54496979
123549926 -34002711
29021302 -63886120
52606548 -87746888
122217966 -28382750
60458246 5337828
71674631 7226027
-127780934 4720715
-47005317 232118
-59477542 74354967
24638462 -48400589
-96175698 -17082209
114081001 -12397548
-63902697 76789745
-84514889 -18163018
115147282 -71558357
-37346631 19335127
84370683 6273652
35784993 -74572771
123917647 -48979013
38648754 -7660535
100532356 -85278273
115926969 -70422030
-90194776 -17981743
-48729700 -1772848
124107711 -38178891
123742858 -50233061
38573380 -7737409
-115338991 72135544
-57219318 72873686
-119492340 68730700
-135762120 32006207
-130770447 53599415
-57581346 73123046
-84110362 81794114
111411351 -76226058
64039523 6228872
-41912316 55845669
24448016 -39031367
102181056 -1190858
-64989440 -13659147
-39343419 50402924
64364200 6295914
-117751459 70249322
-46617391 714126
45938692 -1535122
36333197 -10184477
-83836509 81784309
81256396 -92219854
119980616 -22345973
-115866187 -8100580
-109809598 75659016
100051917 -85570440
123320041 -32789543
-85069737 81816617
71428257 -92621036
121654753 -26618562
62358741 5845629
-115584499 -8311440
-132599465 14320126
-135758679 32436440
117567288 -67785392
-37860469 17489663
28409081 -22884205
-44250080 3951679
93361872 -88933505
96914926 -87304203
-35932391 27702665
-92572848 81355430
-53058982 -6000304
-122996794 -1549297
122158269 -57224075
-113456847 73450985
107393897 -80196311
32439858 -15379493
-40345464 52733048
-85668750 81821328
34802544 -73346276
-58343492 -9990067
78765328 7097048
117425453 -17380216
-116330211 71389240
-40919832 53937894
103994261 -82941347
-82120874 -18045797
-35975637 27209250
-93106112 -17636352
122293745 -28638716
93481003 -88884237
-96070819 -17104424
94675598 -88370079
-134385204 20166849
62393975 5854291
124240309 -40010519
-131642616 11946522
123594910 -51147434
106826308 -80689580
41197361 -5242881
-119062812 69118689
-35896932 35485965
-117020185 70846468
24313748 -42417937
-52199488 -5239707
34089398 -13004694
-96004400 80761202
114271202 -12648765
80640068 -92302376
110980595 -8711934
-42324434 7059797
-45155937 60995209
-58023123 73421044
64832438 6388658
-65732601 77634129
-36860372 42241860
-133167931 15924200
86165887 5870442
-68862991 78878911
-73110784 80194291
-88685620 81735894
90269340 4681258
-39638695 12518614
27033040 -58971223
27313188 -25644879
-93659495 -17551015
51594026 -87244766
-126053665 61429167
-86561801 -18172195
-135753031 30850332
32705747 -14976309
-58801133 73929452
36056250 -10510738
77475840 -92604178
24604177 -48093205
122733015 -30234550
106767253 -4668588
-46533223 820338
120987999 -24775636
-57533686 -9447656
30552699 -18516668
-135049998 40232050
39896868 -6434015
-64902248 -13619222
95735813 -87882410
82885450 6555319
88437777 -90667728
65239608 -91874176
26885625 -58536280
26324399 -56743590
-37026556 42993322
99863241 -85683168
-135523171 26935536
-59951317 -11001298
80606218 6898045
53026191 2538083
-51570338 -4660069
-40014768 11643493
112476858 -10400845
-91268427 -17874503
-45885324 1658294
38853605 -7453269
110680420 -77017661
-38595783 15229116
-104559245 78153704
29892605 -19751434
123406893 -33228955
88932481 5110623
-57617081 -9504576
37697597 -8655565
111078226 -76591084
69785653 7090298
67973830 6892212
-114095127 73019232
41901124 -80776290
-104260278 78273873
97974653 -86751143
26659010 -27568130
-87981513 81772149
-36221769 25059776
36331070 -75221675
-74804578 -16963057
62570159 5897198
-43388423 58362721
-135460315 26332937
-126947284 3469865
119753838 -21844701
-83744617 -18137851
79649252 -92418739
-135458128 37329661
80614704 -92305605
-81075112 81601218
122061871 -27870627
101671153 -84555573
-109320443 75923813
-98466397 -16537762
51270802 1669778
59411871 5023645
-89685147 -18024472
-129100463 56757686
-38224533 16324790
111734651 -9543881
-76373912 80932057
69814522 -92501551
88702072 -90589102
29066927 -63983403
-135743214 30434197
24876045 -35221363
122023708 -57660302
70976132 -92592852
-37456952 44727914
-64196725 76931492
-35847613 28904592
-114446653 -9132426
-57567378 -9470682
73256867 7284528
-36284236 39025940
48509113 -85531449
55916664 -89197087
117202766 -17004868
-104440085 -14559037
-132977177 15366305
-100993891 79444976
30131608 -19294684
-39024355 49585127
-113671807 -9664346
-123057499 -1481440
98162332 -86649807
-105162771 -14261423
-59069449 -10457221
67625811 6846489
-51363638 -4465234
-51590049 68321364
-77721554 81170618
-41985652 55979042
109939638 -7622450
-36267405 24733439
123411735 -33254074
-126471098 60852553
-36666326 41285529
-42611035 6563656
-109599707 75773500
41209328 -5232306
44741978 -83022540
124125754 -38382009
-57999551 -9762485
-74042157 80428354
-53273922 69827926
-128809476 6386114
-69566015 79125487
-46074744 62233338
-135704599 34226172
123229419 -53056942
-60539483 -11350351
108899647 -78811920
-36542873 23021862
27873471 -24173887
94679829 -88368192
52394321 2235360
-132962906 15325416
62558667 -91303057
117469563 -17455461
123149186 -53429049
-62982731 -12687953
73885271 7293853
34556129 -73025877
-133401276 47005070
102117984 -1148587
-72148602 79932359
100122820 119150
-114892470 72459361
-116438643 71305234
-72302457 -16332798
-135726792 29933103
-129969689 8460743
-64904315 -13620170
33973715 -13162011
43306538 -3479278
-93954288 -17502960
-35767335 31123308
30442555 -66691615
108193623 -5933427
-101209818 -15732545
91342336 4306311
79246820 -92460290
-36033761 37022989
33421053 -13931856
-85903549 -18178385
114324521 -12719783
124050479 -47820563
-36517937 40480332
-37390164 19167528
28477608 -22726242
-133710327 45998939
-133193777 47641349
-35768044 31074282
-110605590 75212878
82531462 6615613
-83524574 81771309
120637982 -61518922
104237386 -82760864
-135760076 31333012
-120678841 67610331
-110223749 75429275
-39606833 12594923
124032288 -47994425
-116420232 71319530
27020762 -58935494
39932159 -79008076
-107054185 77061470
49290524 -85992646
112750495 -74681708
-135728845 29988242
40254083 -79310262
-101652083 -15586600
27844462 -24246753
25491353 -31912938
-133937838 18437356
29455847 -20617166
59563906 -90479483
74600144 -92703494
-49188584 65914692
-122315149 -2294579
-39674011 51208388
31227236 -68074049
119347645 -20981653
-107374864 76909153
81201383 -92227537
50971347 1512993
117512920 -17529714
124298419 -43908757
78963990 -92487525
-130547252 54054780
31118074 -17520900
-134874061 22437878
109700517 -7381260
113063995 -11107566
-42731456 6359040
-74514819 80539898
85361431 6059711
31818462 -16356256
93107844 3628727
84912142 6159335
65661282 -91950121
27966805 -23941672
116999132 -16668037
-135727099 33701563
101718135 -84524823
89074179 -90475706
32395017 -15448332
47078984 -84636554
25032872 -51158672
-39610202 12586839
35554374 -74291888
-64018703 -13203122
121137914 -25170903
24761216 -49384732
-113109997 -10036825
24358948 -40559564
-119458476 -5118787
-37867948 17464691
-71851115 79847184
-135052260 23424049
65366631 -91897446
113807095 -12041463
91907102 -89506509
-71323824 -16048468
-134840878 41376993
121941045 -27487143
-129100812 6885750
-126217571 61204814
27236562 -59551152
-135310604 25114587
-100830194 79497022
38826489 -7480567
-58699953 -10221665
-56114906 -8440508
31669940 -68811322
-51416905 68158487
115610337 -70891620
104989786 -3221027
-51917913 68625580
76689873 7239151
124028630 -48028693
-115900630 -8074584
-127333108 4038662
80429996 6920085
-135315279 25149212
40385067 -5976941
96677682 -87423640
53645849 -88233121
73532857 -92698233
124311035 -43139366
124189832 -46210662
-123636153 -821861
24660448 -48589152
-64738113 -13543514
-85301495 81819291
48892630 351558
-49807547 -2924004
27159581 -59334469
-123179644 -1344146
36574331 -9904954
58659334 -90190790
47677401 -85019195
106557655 -80917897
41133755 -80109432
-105927872 77574625
96917844 -87302724
-112549058 -10397991
121193749 -60087890
123802802 -49828382
27837854 -61145238
-130038686 8591780
-91399822 81502601
28454671 -62629688
-52581914 69225309
-129644101 55788094
124306217 -41885168
-94263469 -17450614
24399871 -45651647
49986762 979061
-112466096 -10450512
-101397683 -15671109
-97859071 -16693242
-90006765 81640949
-135741295 30366707
71024865 -92596088
25528734 -53663446
72226209 7252119
45489256 -1848523
25237212 -33134879
-40480136 53023091
109551473 -78176067
-121607764 66679905
-37208916 19881850
-66724323 78055053
24473925 -38695096
114166133 -12509583
-108446529 -12736783
25643108 -31247939
122298030 -56755293
-75280732 -17067575
51729192 1904816
79688970 -92414460
-131337088 52387247
46914855 -84529498
-38431399 15704019
-105890232 77591197
-72272556 -16324430
-119997755 -4619560
116319290 -15584851
123647867 -34576360
35084574 -11702120
30682140 -67124688
-82073001 -18042278
-90551733 -17948685
25595808 -53957862
108531471 -6247614
-132157917 13181091
57551265 4402440
84141490 -91728796
123551993 -34014417
117941089 -67129378
-49143196 -2223105
56223976 -89317512
120129494 -22683275
27968932 -23936417
112825808 -74590962
-80539863 -17905081
-80329417 -17882528
53762523 -88285903
63620376 6138996
27973520 -23925092
98778988 -86309500
123387095 -33126945
-62000746 75814380
-129609605 7791683
122695874 -55317425
-53950877 70396318
120359829 -62189770
53402958 -88122081
46313774 -84129521
-80877127 81582178
89567098 -90320600
36892618 -75866351
-35766443 31190437
-35864205 28633319
-134429229 43290755
69971475 7106843
-134425942 20338136
47588807 -445338
-38574217 48352283
-107608585 76796374
-50020550 66785908
108384610 -79298313
97948997 1356279
-108979603 -12461362
24371070 -40293736
-135758633 31202654
-121631992 -3012143
121960012 -57862002
-128995557 6703703
25636161 -54131149
-43117537 57925255
101023000 -84972179
97331594 1681952
-108091621 76558521
77207747 -92620448
-40753463 53597322
97210289 1744651
66252376 -92050316
-40953410 54005845
-80437334 81537037
37646459 -8710641
-134765086 41758355
24518136 -38178974
33163143 -71106699
-135235995 39058093
31603979 -16705362
-79306138 81402822
123752246 -50171166
-87181555 81801273
-130284147 9065767
-77552145 81142716
-127403194 4143818
-131159464 10866412
-48458135 -1471035
