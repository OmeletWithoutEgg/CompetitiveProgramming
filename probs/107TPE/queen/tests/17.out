-5726262.01 -5441665.87
